iscript/src/iscript/mac.py

```diff
@@ -1308,17 +1294,47 @@ def get_notarization_status_from_log(log_path):
 
     """
     key_config = get_key_config(config, task, base_key="mac_config")
+    work_dir = config["work_dir"]
 
     all_paths = get_app_paths(config, task)
-    langpack_apps = filter_apps(all_paths, fmt="autograph_langpack")
```

Comment:
no langpacks in geckodriver. this was added afterwards, and afaict is extraneous.